code/game/objects/items/kirbyplants.dm

```diff
@@ -0,0 +1,103 @@
+
+/obj/item/kirbyplants
+	name = "potted plant"
+	icon = 'icons/obj/flora/plants.dmi'
+	icon_state = "plant-01"
+	desc = "A little bit of nature contained in a pot."
+	layer = ABOVE_MOB_LAYER
+	plane = GAME_PLANE_UPPER
+	w_class = WEIGHT_CLASS_HUGE
+	force = 10
+	throwforce = 13
+	throw_speed = 2
+	throw_range = 4
+	item_flags = NO_PIXEL_RANDOM_DROP
+
+	/// Can this plant be trimmed by someone with TRAIT_BONSAI
+	var/trimmable = TRUE
+	var/list/static/random_plant_states
+
+/obj/item/kirbyplants/ComponentInitialize()
```

Comment:
ComponentInitialize() is cringe dont use it for new things. use Initialize() instead

Reply:
I moved the old code into a new file, I touched nothing of that. These are the potted plants you see throughout the station